.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
We don't commit such workspace files. It is at user discretion to use what method he may like for testing.

If you really want this to be published then I would say write a document/wiki for how to run tests via vscode and example env files.